src/downshift.js

```diff
@@ -172,6 +173,13 @@ class Downshift extends Component {
   }
 
   getItemNodeFromIndex = index => {
+    if (isReactNative()) {
+      throw new Error(
```

Comment:
We don't need to worry about throwing an error because `getItemNodeFromIndex` is not publicly exposed so this should never be called :)

Reply:
Heh, good call. This was part of my debugging earlier to figure out what codepath was being followed to call this. I'll remove it!